src/components/ShoppingCart/dropDownButton.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { GrDown, GrUp } from "react-icons/gr";
+import { IconButton } from "@chakra-ui/react";
+import { Collapse } from "@chakra-ui/react";
+import { PriceBar } from "@/store/reducer/priceBarSlice";
+
+type detail = {
+  detail: PriceBar[];
+};
+
+const DropDownButton = ({ detail }: detail) => {
+  const [show, setShow] = useState(false);
+
+  const handleToggle = () => setShow(!show);
+
+  const scroll = "scroll";
```

Comment:
In my opinion, you do not need to define these variables. Instead, you can simply use these strings as the chakra props.

Reply:
Thank you for your advice. I have fixed that.